github/repos.go

```diff
@@ -529,6 +531,12 @@ type RequiredStatusChecks struct {
 	Contexts *[]string `json:"contexts,omitempty"`
 }
 
+// RequiredPullRequestReviews represents the protection configuration for pull requests.
+type RequiredPullRequestReviews struct {
+	// Enforce pull request reviews for repository administrators.
+	IncludeAdmins *bool `json:"include_admins,omitempty"`
```

Comment:
Yeah, I saw that similar fields above that are required also have omitempty, and I figured you probably just followed the pattern. I wanted to bring it up so we could discuss the best thing to do from now on.

I do think leaving out `omitempty` would be a more appropriate thing to do, and it's better to start doing it late than never. There are at least a few places where it's already done, e.g., see here:

- https://github.com/google/go-github/blob/dcda0b96bd591fbeb40dcaddec3fb40a7fcaca34/github/projects.go#L169-L170
- https://github.com/google/go-github/blob/dcda0b96bd591fbeb40dcaddec3fb40a7fcaca34/github/pulls.go#L291

What if we go a step further to increase the mapping between a required field and it always being included by making it a value rather than pointer? E.g.:

```Go
IncludeAdmins bool `json:"include_admins"` // Required.
```